src/schema/v2/image/index.ts

```diff
@@ -61,8 +61,8 @@ export const ImageType = new GraphQLObjectType<any, ResolverContext>({
       type: GraphQLString,
     },
     height: {
-      type: GraphQLInt,
-      resolve: ({ original_height }) => original_height,
+      type: new GraphQLNonNull(GraphQLInt),
```

Comment:
non-null constraint _with_ a fallback in MP?! I'm sold.

Reply:
Well here's the problem though, now that I'm thinking through this a bit: Now folks won't handle the `0` dimension case.

Unfortunately I think I really need to make the opposite change. Or what I can do is just alias the fields. It's all super ugly either way.